src/Action.ts

```diff
@@ -38,6 +38,9 @@ export class ActionBase {
   createdDateTime: string
   payload: string
   isTerminal: boolean
+  // If true, CL will generate out of domain example utterances that point to this action.
+  // There should be at most one action in a model with this flag set to `true`.
```

Comment:
Will you be adding a check for this in the UI code?

Reply:
Can do!

Reply:
thought that we are not exposing this in UI and this is only used for .dialog import.

Reply:
In short term yes.  However weird to have validity check only mentioned in a comment.  There should at least be another comment in the action creator that this needs to be checked